Readme.md

```diff
@@ -234,6 +234,15 @@ I will follow my element movements
 </a>
 ```
 
+####Tooltip hidden
+I case you want to keep the tooltips hidden for some reasons, the `tooltip-hidden`argument can be set. In case you want to hide the tooltips and disable the `tooltip-show-trigger` the `tooltip-hidden` attribute must be assigned to `tooltip-hidden="true"`. To switch the tooltip behaviour dynamically, assign the `tooltip-hidden`attribute to a variable or function with type of boolean.
+Default: `false`
+```html
+<a href="#" tooltips tooltip-title="Turn me on or off" tooltip-hidden="{{tooltipHidden}}">
```

Comment:
Hi, as soon as the PR is merged it will be in 'master' branch.

Then you'll be able to use bower or npm to update the module version and get latest one.